tool/inits/ELBDM_RESCALE_PERIODIC_IC.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python
+
+import numpy as np
+import scipy
+import sys
+import argparse
+
+# load the command-line parameters
+parser = argparse.ArgumentParser( description='Upscale periodic GAMER ELBDM initial conditions. Example usage: ./ELBDM_RESCALE_PERIODIC_IC.py -n_in 256 -n_out 64 -input UM_IC_high_resolution -output UM_IC_low_resolution' )
```

Comment:
Replace "Upscale" with "Rescale" if the input resolution can indeed be higher than the output resolution (as adopted in the provided example). Otherwise, revise the example.